.github/ISSUE_TEMPLATE/01-bug-report.yml

```diff
@@ -0,0 +1,18 @@
+name: Bug Report
+description: Create a report to help us improve
+labels: ["bug"]
+body: 
+- id: description
+  type: textarea
+  attributes:
+    label: Description
+    placeholder: Please provide a succinct description of the issue.
```

Comment:
How can we tell the user to include information such as :

1. Package versions (worker, worker.sdk , worker extensions if any)
2. Repro steps
3. Link to minimal repro app (if possible to provide)
4. If issue only happening on deployed environment, provide SKU details

Reply:
I agree with all of the above. Let's get the changes in and iterate.